### DISCUSSION TITLE
Logical OR assignment operator (" ||= ") not supported?

### DISCUSSION THREAD

**Discussion by BennyAlex** [oven-sh/bun#10634] [Q&A] [answered]
i am using the husky libary and it uses the ||= operator. When starting the bun server, I see this error:

![image](https://github.com/oven-sh/bun/assets/17402724/4d935459-970b-4f2f-b723-93c538f21e83)

**Comment by Jarred-Sumner** [comment, marked as answer]
Bun supports `||=` and the other modern ECMAScript features you'd expect. 

This script is running with an old version of Node.js. `./start.sh` is either running `node` or a script it starts is running `node`. Either update the `start.sh` script to run `bun`, or prefix the invocation with `--bun`, like `bun --bun start`

**Reply by BennyAlex** [reply]
I dont think it is running by node. In the husky source code I cannot see any node / npm calls either.

Thats the start.sh
```
#!/bin/bash

# Assign the job name from either the RUN_JOB environment variable or the first command line argument
jobName=${RUN_JOB:-$1}

# Assign the job ID from either the JOB_ID environment variable or the second command line argument
jobid=${JOB_ID:-$2}

# Check if both jobName and jobid are not empty
if [ -n "$jobName" ] && [ -n "$jobid" ]; then
    echo "Starting job: $jobName with jobid: $jobid"
    # Run the job with the specified jobid
    bun run src/jobs/"$jobName".ts "$jobid"
elif [ -n "$jobName" ] || [ -n "$jobid" ]; then
    # Error out if only one of jobName or jobid is provided
    echo "Error: Both job name and job ID must be provided."
    exit 1
else
    # If both jobName and jobid are empty, start the server
    echo "No job name or job ID provided, starting server."
    bun run src/startServer.ts
fi

```

**Comment by Jarred-Sumner** [comment]
Husky itself runs node I think
